material-overrides/assets/stylesheets/moonbeam.css

```diff
@@ -1178,6 +1189,28 @@ html .md-typeset details.function div.tabbed-set,
 .md-feedback__icon:last-child:hover {
   background: rgba(226, 101, 96, 0.35);
 }
+/*divider between feedback and edit this button*/
+.divider {
+  width: 100%;
+  height: 1px;
+  background-color: var(--purple-dark-tint-20);
+  margin: 0.75em;
+}
+
+/*styles for the edit this page button*/
+.md-content__button {
+  margin: 0.75em auto 0 !important;
```

Comment:
You shouldn't need to use `!important`. Since you're trying to override a styling it looks like, you need to make sure you're targeting the right class to apply the margin to